pages/conduct.md

```diff
@@ -0,0 +1,40 @@
+<script>{
+	"title": "jQuery Foundation Code of Conduct",
+	"pageTemplate": "page-conduct.php"
+}</script>
+
+Like the technical community as a whole, the jQuery team and community is made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the jQuery project or jQuery Foundation. This includes IRC, the mailing lists, the issue tracker, jQuery events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```

Comment:
I'm pretty sure most places that say jQuery should say jQuery Foundation. In this example, I think it would make more sense to say "managed by a jQuery Foundation project or the jQuery Foundation."